docs/accessibility/accessibility-statement.md

```diff
@@ -13,15 +13,23 @@ eleventyNavigation:
+
+The Office of Technology Services (O.T.S.), and Pelican, Louisiana’s Design System, are committed to providing digital products that are accessible. We are actively working to increase the accessibility and usability of our digital products.
+
+## Your Role in Accessibility
+
+O.T.S. works to create a culture which cares about accessibility. Part of caring means developing the skills to help create accessible products. O.T.S. has also created <a href="https://accessibility.ots.la.gov" target="_blank">Accessible Web Services</a> to help create a caring culture. This resource presents our roles towards creating accessible products. There are also numerous resource links to help grow your skills for accessibility.
+
+While Pelican strives to create the starting point for accessible products, be sure to build accessibility into your project to serve your users.
 
 ## Accessibility is a Human Right
 
-Accessibility isn’t a line-item that is tacked on at the end of a digital product’s development process. It's a continual part of the development process. It has to be implemented along with content development, UX, wireframe, design, and development choices. According to a <a href="https://www.census.gov/content/dam/Census/library/publications/2018/demo/p70-152.pdf" target="_blank">2014 Census report on Americans with disabilities</a>, “...27.2 percent, or 85.3 million, of people living in the United States had a disability in 2014.” This means that an accessibility-first mindset should the prime factor in developing digital products for Louisiana.
+According to a <a href="https://www.census.gov/content/dam/Census/library/publications/2018/demo/p70-152.pdf" target="_blank">2014 Census report on Americans with disabilities</a>, “...27.2 percent, or 85.3 million, of people living in the United States had a disability in 2014.” This means that an accessibility-first mindset should be an important part of developing digital products for Louisiana.
 
 ## Ongoing Committment
 
-Pelican endeavours to conform to level Double-A of the World Wide Web Consortium (W3C) <a href="http://www.w3.org/TR/WCAG20/" target="_blank">Web Content Accessibility Guidelines 2.</a>.<a href="https://www.w3.org/TR/WCAG21/" target="_blank">1</a>. These guidelines explain how to make web content more accessible for people with disabilities. Conformance with these guidelines will help make the web more user friendly for all people.
+Pelican endeavours to conform to level A.A. of the World Wide Web Consortium (W3C) <a href="http://www.w3.org/TR/WCAG20/" target="_blank">Web Content Accessibility Guidelines 2.</a>.<a href="https://www.w3.org/TR/WCAG21/" target="_blank">1</a>. These guidelines explain how to make web content more accessible for people with disabilities. Conformance with these guidelines will help make the web more user friendly for all people.
```

Comment:
This could be updated as well with the markdown way to right this, but I question the accessibility of splitting the link up like this. I assume it will render wonky with the new site icon after the first part, and another after the 1. I suggest re-writing and being intentional with both of these.